spec/factories/organizations.rb

```diff
@@ -2,10 +2,28 @@
   factory :organization do
     name { Faker::Company.unique.name }
 
-    factory :organization_with_device_group do
+    trait :has_one_device_group do
       after(:create) do |organization|
         create(:device_group, organization: organization)
       end
     end
+
+    trait :has_many_device_groups do
+      after(:create) do |organization|
+        create_list(:device_group, 5, organization: organization)
```

Comment:
why 5?
check w/ the [docs](https://github.com/thoughtbot/factory_bot/blob/master/GETTING_STARTED.md#transient-attributes) and try to make something similar here

Reply:
[here](https://github.com/thoughtbot/factory_bot/blob/master/GETTING_STARTED.md#associations) is an examples w/ associations